encrypted-media-respec.html

```diff
@@ -2089,34 +2078,48 @@
           </ol>
         </section>
 
-        <section id="session-close">
-          <h4>Session Close</h4>
-          <p>The Session Close algorithm is run when the CDM closes the session associated with a <a>MediaKeySession</a> object.</p>
+        <section id="close-session">
+          <h4>Close Session</h4>
+          <p>The Close Session algorithm closes the session associated with a <a>MediaKeySession</a> object.</p>
           <p>Closing a session means that the license(s) and key(s) associated with it are no longer available to decrypt <a def-id="media-data"></a>.
             All <a>MediaKeySession</a> methods will fail for this object.
           </p>
-          <p class="note">The CDM may close a session at any point, such as in response to a <a def-id="close"></a> call, when the session is no longer needed, or when system resources are lost.
-          Keys in other sessions SHOULD be unaffected, even if they have overlapping key IDs.
+          <p>The following steps are run:</p>
+          <ol>
+            <li><p>Let <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
+            <li><p>Let <var>cdm</var> be the CDM instance represented by <var>session</var>'s <var>cdm instance</var> value.</p></li>
+            <li>
+              <p>Use <var>cdm</var> to close the <a href="#key-session">session</a> associated with <var>session</var>.</p>
```

Comment:
Previously:

1. `close()` initiated a request to close the session
1. The session may not have been closed.
1. There was normative text to "not remove stored session data. Do not generate or send key or license release messages."

I don't recall the reasons for (1) and (2), and it seems reasonable to require that the session be close. (3) seems important. See also the comment on the next line.